apps/test/unit/gamelab/AnimationPicker/animationPickerModuleTest.js

```diff
@@ -102,5 +105,25 @@ describe('animationPickerModule', function () {
         expect(newState.uploadError).to.equal(status);
       });
     });
+
+    describe('action: handleUploadComplete', function () {
+
+      it('sets sourceUrl to null', function () {
+        const newState = {animationPicker: { uploadFilename: "filename.jpg", goal: Goal.NEW_ANIMATION }};
+        var store = createStore(combineReducers({animationList: listReducer, animationPicker: reducer}), newState);
+
+        var onMetadataLoaded = animationPickerModule.buildOnImageMetadataLoaded(
+          "filename.jpg", Goal.NEW_ANIMATION, "filename.jpg",
+          {filename: "filename.jpg", result: 0, versionId: "string"}, store.dispatch);
```

Comment:
Style nit: It may seem verbose, but when we have to wrap arguments like this we often just place one argument per line.
```js
var onMetadataLoaded = animationPickerModule.buildOnMetadataLoaded(
  "filename.jpg",
  Goal.NEW_ANIMATION,
  "filename.jpg",
  {
    filename: "filename.jpg",
    result: 0,
    versionId: "string"
  },
  store.dispatch
);
```